### ISSUE TITLE
labels in front does not work with torque maps

### ISSUE THREAD

**Issue by andy-esch** [CartoDB/cartoframes#236]
This produces a map with no labels while the non-time equivalent gives expected behavior.
```python
cc.map(layers=[BaseMap('dark', labels='front'),
               Layer('all_month_3', color='net', time='time')])
```

**Comment by andy-esch** [comment]
This is because we only read the torque tiles from the named map, not anything about the raster layers.

**Comment by andy-esch** [comment]
Fixed in #415 
<img width="926" alt="screen shot 2018-04-06 at 10 19 19 am" src="https://user-images.githubusercontent.com/1041056/38434453-5cf8df56-399c-11e8-8a19-d2f8ca726ec9.png">
